packages/devtools_app/lib/src/screens/vm_developer/object_inspector_view.dart

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '../../shared/split.dart';
+import '../debugger/program_explorer.dart';
+import '../debugger/program_explorer_controller.dart';
+import '../debugger/program_explorer_model.dart';
+import 'object_viewport.dart';
+import 'vm_developer_tools_screen.dart';
+
+/// Displays a program explorer used to select and display information about objects in the Dart VM
+class ObjectInspectorView extends VMDeveloperView {
+  ObjectInspectorView()
+      : super(
+          id,
+          title: 'Objects',
+          icon: Icons.data_object,
+        );
+  static const id = 'inspector-view';
+
+  final programExplorerController = ProgramExplorerController();
+
+  ObjectHistory objectHistory = ObjectHistory();
```

Comment:
Nit: if we don't reassign to a property, we should make it final and drop the left hand side type.

```dart
final objectHistory = ObjectHistory();
```